File: Week 1 - C and 0 - Scratch/hello/hello.c
#include <stdio.h>
#include <cs50.h> // Needed to use get_string function

int main(void)
{
    string name = get_string("What is your name? "); // Asking user what their name is

    printf("hello, %s\n", name); // Printing out "hello ++ user name"
}

File: Week 6 - Python/sentimental-hello/hello.py
# TODO

from cs50 import get_string

# Uses CS50's getstring function to ask user for a name
name = get_string("What is your name? ")

# Prints hello,name to the terminal using much more succinct code than C: printf("Hello, %s/n", name)
print(f"hello, {name}")